CONTRIBUTING.MD

```diff
@@ -2,10 +2,15 @@
 
 You can contribute in many ways: openning or participating of a issue, reporting a bug, sending a [pull request](#submitting-a-pull-request).
 
+## Contribution Prerequisites
+
+- Node installed at v12.16.1 and Yarn.
```

Comment:
here, we should put 
"Node installed at v12.16.1 and Yarn. (to make it easier, we have .nvmrc, so just use `nvm use` to sets the repo default version of node)"

What do you think?